### PR TITLE
Add some extra options to the vagrant inventory file and remove an un…

### PR DESCRIPTION
**Pull request by bmac**
…used web-platform-test.live inventory file

### CODE REVIEW

File: inventory/development [hunk 1]
```diff
@@ -1 +1,2 @@
-vagrant ansible_ssh_host=172.30.1.5 ansible_ssh_user=vagrant
+vagrant ansible_ssh_host=172.30.1.5 ansible_ssh_user=vagrant ansible_ssh_private_key_file=.vagrant/machines/default/virtualbox/private_key ansible_ssh_common_args='-o StrictHostKeyChecking=no'
```
Comment (jugglinmike) [inline inventory/development]: This hasn't been necessary for me to date. When do we need this configuration?

Comment (bmac) [reply inventory/development]: We don't need it when running `vagrant up` but it seems we do need if if we want to do `$ ansible-playbook playbook.yml  -i inventory/development` to update the vagrant box after its been provisioned.

Comment (bmac) [reply inventory/development]: I saw in your documentation commits the suggestion to use `vagrant provision`. I wonder if I should just delete this inventory file too?

Comment (jugglinmike) [reply inventory/development]: Even better!

**Comment by bmac** [comment]
@jugglinmike this pr has been updated and is ready for another review.

**Comment by jugglinmike** [comment]
Thanks, @bmac!

**Pull request merged**